src/boa.ts

```diff
@@ -418,9 +434,13 @@ export async function resubmitJob(uri: vscode.Uri|BoaJob) {
             const jobId = getJobId(uri);
             const job = await client.getJob(jobId);
 
-            await job.resubmit();
-            vscode.commands.executeCommand('boalang.joblist.refresh');
-            vscode.window.showInformationMessage(`Job ${jobId} has been resubmitted`);
+            if (!job.running) {
+                await job.resubmit();
+                vscode.commands.executeCommand('boalang.joblist.refresh');
+                vscode.window.showInformationMessage(`Job ${jobId} has been resubmitted`);
+            } else {
+                vscode.window.showInformationMessage(`Job ${jobId} is running`);
```

Comment:
Minor wording change.

```suggestion
                vscode.window.showInformationMessage(`Job ${jobId} is already running, stop it first`);
```